package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "_s",
+  "version": "1.0.0",
+  "scripts": {
+    "css-build": "npm run css-lint && npm run css-compile && npm run css-prefix && npm run css-format && npm run css-fix",
```

Comment:
I don't think we need to run `css-lint` at the start of the build process. If we were to run a lint it should be `scss-lint`